### PR TITLE
Add inputs

### PR DESCRIPTION
**Pull request by rubenmoya**
#43
----

- [x] Implementation
- [x] Design Aproval
- [x] UI Test
- [x] CR

### CODE REVIEW

File: packages/styles/src/inputs/_inputs.scss [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  min-height: 32px;
+  padding: 6px 8px;
+  border: 0;
+  border-radius: 4px;
+  background: var(--as-color-ui-01, #{$color-ui-01});
+  box-shadow: inset 0 0 0 1px var(--as-color-ui-04, #{$color-ui-04});
+  color: var(--as-color-type-01, #{$color-type-01});
+  font: $font-body;
+
+  &::-webkit-input-placeholder {
```
Comment (IagoLast) [inline packages/styles/src/inputs/_inputs.scss]: can we delegate this browser-specific selectors in a postcss plugin?

File: packages/styles/src/inputs/_inputs.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
+@import "../core/_dev";
+
+.as-input,
+.as-textarea {
+  box-sizing: border-box;
+  width: 100%;
+  min-height: 32px;
+  padding: 6px 8px;
```
Comment (IagoLast) [inline packages/styles/src/inputs/_inputs.scss]: should we use relative rules?

File: packages/styles/src/inputs/_inputs.scss (hunk 1 above)
Comment (rubenmoya) [reply packages/styles/src/inputs/_inputs.scss]: Great idea! It seems like we can use `::placeholder` and it takes care of it :D

screen shot 2018-08-02 at 13 04 45

**Comment by piensaenpixel** [comment]
Could we add a transition (as we have in the radio buttons) in the focus state?

**Comment by piensaenpixel** [comment]
The rest is perfect!

**Pull request merged**